cuenca/http/client.py

```diff
@@ -135,7 +138,9 @@ def request(
             **kwargs,
         )
         self._check_response(resp)
-        return resp.json()
+        if resp.headers['Content-Type'] == 'application/json':
+            return resp.json()
+        return dict(data=resp.content)
```

Comment:
Doesn't this break the interface in other places?

Reply:
It doesn't break it, I even did it this way to avoid breaking the rest.